include/knowrob/terms/Atomic.h

```diff
@@ -57,6 +58,17 @@ namespace knowrob {
 		 */
 		virtual std::string_view stringForm() const = 0;
 
+		/**
+		 * Get the lexical form of this atomic term.
+		 * @return the lexical form of this atomic term.
+		 */
+		std::string humanReadableForm() const {
```

Comment:
Any reason you did not add this to `Term` instead? IMO it should be a method of Term